tests-examples/demo-todo-app.spec.js

```diff
@@ -0,0 +1,449 @@
+// @ts-check
```

Comment:
Same for this file, looks like it was added by playwright by default or something?